### PR TITLE
Implement reverse mode

### PR DESCRIPTION
**Pull request by eugulixes**

### CODE REVIEW

File: gits/terminal.py [hunk 1]
```diff
@@ -669,6 +672,10 @@ def dumphtml(self):
             if self._screen[i] & (1 << UNDERLINE_BIT):
                 current_classes.append('underline')
 
+            if self._screen[i] & (1 << REVERSE_BIT):
```
Comment (dshil) [inline gits/terminal.py]: Are you sure that this if statement will ever be true?
We clean the `REVERSE_BIT` at the beginning of the `dumphtml` method.

Comment (eugulixes) [reply gits/terminal.py]: @dshil, yes, because I clean `_sgr` but check `_screen[i]`.

Comment (dshil) [reply gits/terminal.py]: oops, I got it, thx

**Review by dshil — APPROVED** [review]
LGTM

**Pull request merged**